src/bosh_azure_cpi/spec/unit/vm_manager/create/shared_stuff.rb

```diff
@@ -53,7 +53,17 @@
       'storage_account_name' => 'dfe03ad623f34d42999e93ca',
       'caching' => 'ReadWrite',
       'load_balancer' => 'fake-lb-name',
+      # 'load_balancer' => {
```

Comment:
Why this commented out code is there?

Reply:
Because I wasn't sure whether it was better to change the shared `let`s to use the new Hash or Array<Hash> format for the `load_balancer` and `application_gateway` config in the `:vm_properties`, or leave them using the older, original String-typed values.

All existing specs are passing either way, but I left the original default values as-is, and added the equivalent new-format values as commented-out alternatives.

What would you like done here?

Reply:
I would suggest to remove the commented out code.

Reply:
Done.